server/src/test/java/org/candlepin/dto/api/v1/OwnerInfoDTOTranslatorTest.java

```diff
@@ -0,0 +1,86 @@
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.dto.api.v1;
+
+import static org.junit.Assert.*;
+
+import org.candlepin.dto.AbstractTranslatorTest;
+import org.candlepin.dto.ModelTranslator;
+import org.candlepin.model.ConsumerType;
+import org.candlepin.model.OwnerInfo;
+
+public class OwnerInfoDTOTranslatorTest extends
+    AbstractTranslatorTest<OwnerInfo, OwnerInfoDTO, OwnerInfoDTOTranslator> {
+
+    OwnerInfoDTOTranslator translator = new OwnerInfoDTOTranslator();
+
+    @Override
+    protected void initModelTranslator(ModelTranslator modelTranslator) {
+        modelTranslator.registerTranslator(this.translator,
+            OwnerInfo.class, OwnerInfoDTO.class);
+    }
+
+    @Override
+    protected OwnerInfoDTOTranslator initObjectTranslator() {
+        return this.translator;
+    }
+
+    @Override
+    protected OwnerInfo initSourceObject() {
+        OwnerInfo ownerInfo = new OwnerInfo();
+        ownerInfo.addTypeTotal(new ConsumerType("test"), 1, 2);
+        ownerInfo.setGuestCount(2);
+        ownerInfo.setPhysicalCount(2);
+        ownerInfo.addToConsumerTypeCountByPool(new ConsumerType("test-1"), 5);
+        ownerInfo.addToEnabledConsumerTypeCountByPool(new ConsumerType("test-2"), 1);
+        ownerInfo.setConsumerCountByComplianceStatus("test-status", 4);
+        ownerInfo.addToEntitlementsConsumedByFamily("test-family", 1, 3);
+        return ownerInfo;
+    }
+
+    @Override
+    protected OwnerInfoDTO initDestinationObject() {
+        return new OwnerInfoDTO();
+    }
+
+    @Override
+    protected void verifyOutput(OwnerInfo source, OwnerInfoDTO dest, boolean childrenGenerated) {
+        if (source != null) {
+            assertEquals(source.getConsumerCounts(), dest.getConsumerCounts());
+            assertEquals(source.getConsumerGuestCounts(), dest.getConsumerGuestCounts());
+            assertEquals(source.getEntitlementsConsumedByType(), dest.getEntitlementsConsumedByType());
+            assertEquals(source.getConsumerTypeCountByPool(), dest.getConsumerTypeCountByPool());
+            assertEquals(source.getEnabledConsumerTypeCountByPool(),
+                dest.getEnabledConsumerTypeCountByPool());
+            assertEquals(source.getConsumerCountsByComplianceStatus(),
+                dest.getConsumerCountsByComplianceStatus());
+            assertEquals(source.getEntitlementsConsumedByFamily().size(),
+                dest.getEntitlementsConsumedByFamily().size());
+
+            //TODO: Add family check here
+            /*boolean isConsumed;
+            if(!source.getEntitlementsConsumedByFamily().isEmpty()) {
+                isConsumed = source.getEntitlementsConsumedByFamily().entrySet().stream()
+                    .allMatch(e -> e.equals(dest.getEntitlementsConsumedByFamily().get(e.getKey())));
+                assertTrue(isConsumed);
+            }*/
```

Comment:
If we end up removing the OwnerInfoTranslator class, then this won't matter, but if we don't, we should either uncomment/update this block or remove it

Reply:
Removed OwnerInfoTranslator.